elastic/beats
# [Metricbeat] gcp.gke: fix overview dashboard

## What does this PR do?


Provide fixes to the Metricbeat GCP GKE Overview dashboard.

## Why is it important?


In the GKE Overview dashboard there were some leftover queries targeting
a cluster name that existed only in the test environment when the
dashboard has been developed.

## Checklist




## Author's Checklist



## How to test this PR locally


Use metricbeat to gather data from a project with GKE clusters and observe the dashboard. All
widgets should be filled.

## Related issues


- Related to #29815

## Use cases



## Screenshots


![Screenshot 2022-01-19 at 15-24-07  Metricbeat GCP  GKE Overview - Elastic](https://user-images.githubusercontent.com/526307/150149935-5a30eb1b-3f17-49a1-b2c4-326f9a106d04.png)

## Logs
